### ISSUE TITLE
Dead kernel because of timeout for Ijulia on Windows 64 bit

### ISSUE THREAD

**Issue by stonebig** [ipython/ipython#7776]
Hi,

I have a quick kernel death (then an automatic restart, then ok) on windows 64bit  with Ipython3.0b1 and Ijulia and Julia0.3.5, with a `timeout` message.

Strange thing is:
- the timeout is pretty quick to come (10 seconds ?)
- I have no issue on 32bit.

Nota: Julia 0.3.x takes up to 40 seconds to warm up on my PC.

==> Maybe a Parameter could be tune ?

![pyjulia_and_lib_openblas](https://cloud.githubusercontent.com/assets/4312421/6190039/7399b400-b39f-11e4-85cc-6c4976fa8455.GIF)

**Comment by minrk** [comment]
The default timeout waiting for the kernel info reply is 10 seconds, but that doesn't mean the kernel is treated as dead. It should behave fine, as long as there's no *other* failure later on. As you can see, the kernel_info reply did arrive 3 seconds after the timeout, so it seems closer to 15 seconds than 40. It's not until 4 seconds later that the kernel dies. The error at the bottom about initializing LinAlg suggests that julia is actually dying due to failing to find openblas, not anything to do with IPython's timeouts.

You can change the timeout with:

    c.NotebookApp.tornado_settings = {'kernel_info_timeout': 60} # default is 10

in your `ipython_notebook_config.py`, but the timeout is unlikely to be related to the kernel dying, here.

**Comment by stonebig** [comment]
The LinAlg was another issue already solved https://github.com/JuliaLang/pyjulia/issues/34

Only the timeout remains for julia0.3.x  64bit.
